styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+.app {
+    width: 100vw;
+    height: 100vh;
```

Comment:
You can remove this width and height from here. It creates scroll bar on the below of the page. 
Instead of width you can give 
max-width:1440px ; width:100%;

Reply:
Actually I think it's alright since it carries the background image. 
considering that it'll fit the view-port